### PR TITLE
Refactor: Log Explorer UI changes. 

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
UI changes: https://linear.app/signoz-io/issue/SIG-523/logs-vqa

### CODE REVIEW

File: frontend/src/components/LogsFormatOptionsMenu/LogsFormatOptionsMenu.styles.scss [hunk 1]
```diff
@@ -88,7 +89,7 @@
 		.max-lines-per-row-input {
 			display: flex;
 
-			border: 1px solid var(--bg-slate-400, #1d212d);
+			// border: 1px solid var(--bg-slate-400, #1d212d);
```
Comment (Vikrant2520) [inline frontend/src/components/LogsFormatOptionsMenu/LogsFormatOptionsMenu.styles.scss:92]: commented code

File: frontend/src/container/LogDetailedView/Overview.styles.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@
 		height: 36px;
 		display: flex;
 		align-items: center;
-		border-top: 1px solid var(--bg-slate-500);
+		// border-top: 1px solid var(--bg-slate-500);
```
Comment (Vikrant2520) [inline frontend/src/container/LogDetailedView/Overview.styles.scss:20]: commented code

File: frontend/src/container/LogsExplorerViews/index.tsx [hunk 3]
```diff
@@ -177,10 +176,8 @@ function LogsExplorerViews({
 		[currentQuery, updateAllQueriesOperators],
 	);
 
-	const handleModeChange = (e: RadioChangeEvent): void => {
-		const panelType = e.target.value;
-
-		if (selectedView === SELECTED_VIEWS.SEARCH) {
+	const handleModeChange = (panelType: PANEL_TYPES): void => {
+		if (selectedView === 'search') {
```
Comment (Vikrant2520) [inline frontend/src/container/LogsExplorerViews/index.tsx:180]: can use the selected view enum here

File: frontend/src/container/QueryBuilder/QueryBuilder.styles.scss [hunk 4]
```diff
@@ -26,7 +26,7 @@
 		left: 16px;
 
 		border-radius: 2px;
-		border: 1px solid var(--bg-slate-200);
+		// border: 1px solid var(--bg-slate-200);
```
Comment (Vikrant2520) [inline frontend/src/container/QueryBuilder/QueryBuilder.styles.scss:29]: commented code

File: frontend/src/components/LogsFormatOptionsMenu/LogsFormatOptionsMenu.styles.scss (hunk 1 above)
Comment (Rajat-Dabade) [reply frontend/src/components/LogsFormatOptionsMenu/LogsFormatOptionsMenu.styles.scss:92]: Done

File: frontend/src/container/LogDetailedView/Overview.styles.scss (hunk 2 above)
Comment (Rajat-Dabade) [reply frontend/src/container/LogDetailedView/Overview.styles.scss:20]: Removed.

File: frontend/src/container/LogsExplorerViews/index.tsx (hunk 3 above)
Comment (Rajat-Dabade) [reply frontend/src/container/LogsExplorerViews/index.tsx:180]: Updated.

**Review by Vikrant2520 — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**